flight/PiOS/Common/pios_servo.c

```diff
@@ -153,6 +153,7 @@ void PIOS_Servo_SetMode(const uint16_t *out_rate, const int banks, const uint16_
 
 		timer_banks[bank].timer = chan->timer;
 		timer_banks[bank].max_pulse = MAX(timer_banks[bank].max_pulse, channel_max[i]);
+		timer_banks[bank].max_pulse = MAX(timer_banks[bank].max_pulse, channel_min[i]);
 	}
```

Comment:
Shouldn't it be 

		timer_banks[bank].max_pulse = MIN(timer_banks[bank].max_pulse, channel_max[i]);
		timer_banks[bank].max_pulse = MAX(timer_banks[bank].max_pulse, channel_min[i]);